script/fs-bench/fio/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM alpine:3.9 AS stage-build
+
+WORKDIR /work
+RUN apk add git build-base linux-headers && \
+    git clone git://git.kernel.dk/fio.git /work/fio && \
+    cd /work/fio && \
+    ./configure --prefix=/work/root && \
+    make && \
+    make install
+
+FROM alpine:3.9 AS stage-final
+
+# How many bytes fio is going to read/write, 512MiB by default.
+ARG size=256M
+
+# How many concurrent reading threads.
+ARG thread=4
+
+WORKDIR /work
+COPY --from=stage-build /work/root/ /
+
+# Lay out test files here, so that it can be compressed into the image layer.
+RUN fio -directory=/work -direct=1 -numjobs=$thread -size=$size -name=test -rw=randread
```

Comment:
`fio` binary is contained in the final stage but maybe, we should provide `fio` binary from outside of the testing container for avoiding the metrics results be affected by lazy-pulling this binary during test (e.g. by mounting kind of `toolbox` directory).

Reply:
`fio` only took 5.8MiB, comparing to the this test 1GiB, it is negligible.

Reply:
@AkihiroSuda WDYT?